kong/clustering/control_plane.lua

```diff
@@ -551,6 +552,11 @@ local function push_config_loop(premature, self, push_config_semaphore, delay)
     if ok then
       if isempty(self.clients) then
         ngx_log(ngx_DEBUG, _log_prefix, "skipping config push (no connected clients)")
+        sleep(1)
```

Comment:
is this sleep needed? can it be `sleep(0)`?  can we be sure that the empty list is still empty after the sleep?

Reply:
It will be nearly a tight loop if we do sleep(0), which may add to the delay of admin API.